server/src/test/java/org/elasticsearch/rest/RestRequestTests.java

```diff
@@ -44,6 +44,7 @@
 import static org.mockito.Mockito.when;
 
 public class RestRequestTests extends ESTestCase {
+    private static final String PARAMETER_KEY = "PARAMETER_KEY";
```

Comment:
This indicates to me that this literal parameter name is important in these tests, but I think that's not the case. Instead, we should use `randomIdentifier()` (within each test, it won't work in static context) to indicate to readers that the parameter name doesn't matter as long as it's equal in all the places it needs to be.